test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -759,6 +759,24 @@ def forward(self, x):
         x = torch.randn(4, 2, 3, requires_grad=True)
         self.run_test(NormModel(), x)
 
+    def test_remainder(self):
+        class RemainderModel(torch.nn.Module):
+            def forward(self, input, other):
+                return torch.remainder(input, other)
+
+        x = torch.randn(4, 2, 3)
+        y = torch.randn(1, 2, 1)
+        self.run_test(RemainderModel(), (x, y))
+
+    def test_remainder_scalar(self):
+        class RemainderModel(torch.nn.Module):
+            def forward(self, input, other):
+                return torch.remainder(input, other)
+
+        x = torch.randint(10, (2, 3)).to(dtype=torch.int)
```

Comment:
Try this example `torch.remainder(torch.tensor(10), 1.5)`

Reply:
This is supported now

Reply:
Added to the tests